app/views/assignments/index.html.erb

```diff
@@ -1,3 +1,5 @@
+
+<%= react_component("ActionItemDashboard", { is_admin: @user.admin?}) %>
```

Comment:
let's also delete all the unused code here (everything besides the react component)